packages/zowe-explorer/src/shared/utils.ts

```diff
@@ -505,8 +505,8 @@ export async function promptForEncoding(node: IZoweDatasetTreeNode | IZoweUSSTre
             });
             if (response != null) {
                 encoding = { kind: "other", codepage: response };
-                const filterEncodingList = encodingHistory.filter((codepage) => codepage !== response);
-                filterEncodingList.unshift(encoding.codepage);
+                const filterEncodingList = encodingHistory.filter((codepage) => codepage.toUpperCase() !== response.toUpperCase());
+                filterEncodingList.unshift(encoding.codepage.toUpperCase());
```

Comment:
really good catch 😋 
Do we want to add a small test for this?

Reply:
Forgot about that, I have just added one. Thank you.

Reply:
@zFernand0 I believe from what we discussed with @t1m0thyj on this setting is that we wanted them to share this setting between MVS and USS, did this change? I originally intended for these to be independent but it seemed that we wanted to maintain them as shared between each other 🤔